core/events/services/classic/models.py

```diff
@@ -67,6 +67,10 @@ class Submission(Base):    # type: ignore
     WITHDRAWN_FORMAT = 'withdrawn'
 
     submission_id = Column(Integer, primary_key=True)
+
+    type = Column(String(8), index=True)
+    """Submission type (e.g. ``new``, ``jref``, ``cross``)."""
```

Comment:
I added some notes to this class for reference, since many of the classic columns are pretty cryptic. There is more to do here, as we go along.